### ISSUE TITLE
Redesign: Mobile navigation full height overlay

### ISSUE THREAD

**Issue by adachiu** [TheOdinProject/theodinproject#495]
The current mobile view menu dropdown covers about 75% of the screen height. This task involves making the mobile view menu dropdown to fill the screen.

Maybe with `height: 100vh;` for mobile view?

Example of full height mobile view menu dropdown: http://talklife.co/

- [ ] If you want to work on this task, please claim it by commenting below   
- [ ] Branch off the redesign branch                                                                 
- [ ] When you are submitting your pull request for this task please ensure the base branch you want to merge into is the redesign branch

**Comment by codyloyd** [comment]
working on this... trying to sort out an inconsistency with the way that `100vh` works.  Not sure what the issue is... but sometimes it overflows the screen.   If I set it to something smaller than 100vh it looks good on some screens, but on other's it's not long enough.

I tried another method to fill the screen, but that broke the animations.   If anyone has any ideas, that would be great, otherwise I'll keep googling and hacking on it 💃
